jenkins-integration/beaker/install/pe/60_classify_nodes.rb

```diff
@@ -0,0 +1,26 @@
+require 'puppet/gatling/config'
+require 'scooter'
+
+test_name "Classify PE agents via Node Classifier"
+
+def classify_pe_nodes(classifier, nodes)
+  production_id = classifier.get_node_group_id_by_name('Production environment')
```

Comment:
I feel like I'm hardcoding this to a UI label rather than some more concrete ID, but this is apparently the best way to refer to this group at the time.